w3mhelperpanel.c

```diff
@@ -167,7 +167,7 @@ int
 main(void)
 {
     Str mailcapfile;
-    extern char *getenv();
+    extern char *getenv(const char *);
```

Comment:
why are we redeclaring things that should be declared in standard headers already :)